src/main/resources/anniv5Resources/localization/eng/farmerpack/Cardstrings.json

```
@@ -0,0 +1,50 @@
+{
+  "${ModID}:Fertilizer": {
+    "NAME": "Fertilizer",
+    "DESCRIPTION": "Draw a card. NL Exhaust."
+  },
+  "${ModID}:HarvestBeans": {
+    "NAME": "Harvest Beans",
+    "DESCRIPTION": "Deal !D! damage. NL If the last card you played this combat wasn't an Attack, deal !D! damage to ALL enemies."
+  },
+  "${ModID}:HarvestCarrots": {
+    "NAME": "Harvest Carrots",
+    "DESCRIPTION": "Gain !B! Block. NL Repeat for each card type you've played this turn.",
+    "EXTENDED_DESCRIPTION": [" NL (Repeats !M! times.)"]
+  },
+  "${ModID}:HarvestPotatoes": {
+    "NAME": "Harvest Potatoes",
+    "DESCRIPTION": "Gain !B! Block. NL If the last card you played this combat wasn't a skill, gain !B! Block again."
```

One more thing that I missed on the earlier iterations -- I'll just make this change.
```suggestion
    "DESCRIPTION": "Gain !B! Block. NL If the last card you played this combat wasn't a Skill, gain !B! Block again."
```